float[] point_to_plane_distance